src/test/kotlin/uk/gov/justice/digital/hmpps/digitalprisonreportingmi/integration/EstablishmentsIntegrationTest.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.digitalprisonreportingmi.integration
+
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import uk.gov.justice.digital.hmpps.digitalprisonreportingmi.data.Establishment
+import uk.gov.justice.digital.hmpps.digitalprisonreportingmi.data.EstablishmentRepository
+import java.util.stream.IntStream
+
+class EstablishmentsIntegrationTest : IntegrationTestBase() {
```

Comment:
Is there any value in providing a test for the case where the table is empty?

Reply:
Just added